externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/schema/RowSet.scala

```diff
@@ -169,12 +169,13 @@ object RowSet {
     var idx = 0
     while (idx < size) {
       val row = rows(idx)
-      val isNull = row.isNullAt(ordinal)
+      val value = row.get(ordinal)
+      val isNull = value == null
```

Comment:
I think we can not make such an assumption, the implementation may be different, for example, the developer uses a custom value to represent `NULL` in its own `CustomRow`, especially for those non-JVM implementations